public/index.jsx

```diff
@@ -8,12 +8,14 @@ import App from '../components/App';
 import reducers from '../reducers';
 
 const createStoreWithMiddleware = applyMiddleware(ReduxPromise)(createStore);
-
+const Store = createStoreWithMiddleware(reducers, {mainView: 'graphAbsolute'})
```

Comment:
I like this, but I think @CaptainNemo7 should take a look to see if this change of store will cause him any issues. I don't see a problem with it for my code and I will def want to add in some lat long defaults to this store for the locationSentiment component and Location container.